### PR TITLE
Delete Sheet changes

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgDeleteSheet.Designer.vb [hunk 1]
```diff
@@ -49,10 +49,11 @@ Partial Class dlgDeleteSheet
         Me.ClientSize = New System.Drawing.Size(419, 127)
         Me.Controls.Add(Me.ucrBase)
         Me.Controls.Add(Me.ucrDataFrameToDelete)
-        Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.SizableToolWindow
+        Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedToolWindow
         Me.MaximizeBox = False
         Me.MinimizeBox = False
         Me.Name = "dlgDeleteSheet"
+        Me.Text = "Delete Sheet"
```
Comment (dannyparsons) [inline instat/dlgDeleteSheet.Designer.vb]: I think this should actually be "Delete Data Frame". Could this be changed and on the menu?

Comment (lilyclements) [inline instat/dlgDeleteSheet.Designer.vb]: Okay sure! It didn't have a name before so I didn't realise, sorry!

Comment (dannyparsons) [inline instat/dlgDeleteSheet.Designer.vb]: I've only just noticed it now, I think we never thought of changing it before

Comment (lilyclements) [inline instat/dlgDeleteSheet.Designer.vb]: It was already "Delete Data Frame" on the main form, I've chnaged it on the dialog text though

**Pull request merged**